src/main.js

```diff
@@ -33,9 +25,18 @@ let load = base.loader({
     }
   },
 
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
+      project: cfg.monitor.component,
```

Comment:
This is somewhat different than what I expected people to do. It will require you to have 6 scopes on the credentials rather than just 2. It will also show up in 3 sentry projects rather than 1. If that's what you expect, this seems fine to me!